syntax = "proto3";

import "google/protobuf/wrappers.proto";

option go_package = "github.com/pomerium/envoy-custom/source/extensions/request_id/uuidx";

package pomerium.extensions;

message UuidxRequestIdConfig {
  google.protobuf.BoolValue pack_trace_reason = 1;
  google.protobuf.BoolValue use_request_id_for_trace_sampling = 2;
}